examples/Cargo.toml

```diff
@@ -27,6 +27,9 @@ rand = "0.8.3"
 [target.'cfg(windows)'.dev-dependencies.windows-sys]
 version = "0.52"
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(tokio_unstable)', 'cfg(tokio_taskdump)'] }
```

Comment:
We usually do it like this:

https://github.com/tokio-rs/tokio/blob/32970527633bb72fc4f01d02523484a9376ac26a/tokio/src/lib.rs#L1

https://github.com/tokio-rs/tokio/blob/32970527633bb72fc4f01d02523484a9376ac26a/.github/workflows/ci.yml#L1050-L1065

It may be worth to switch, but for now I would add a `#[allow(unexpected_cfgs)]` in the source code.